### PR TITLE
Milestone #1 Project Setup

### PR DESCRIPTION
**Pull request by Elukoye**
Please review if the project setup is correct
Thanks.

### CODE REVIEW

File: tic-tac/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+linters:
+   rubocop:
+       display_cop_names: true
+       fixer: true
```
Comment (bolah2009) [inline tic-tac/.rubocop.yml]: ### `REQUIRED`

The `.rubocop.yml` is having the wrong content. Use this instead. You can find the file [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)

```yml
AllCops:
  Exclude:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
  DisplayCopNames: true

Metrics/LineLength:
  Max: 120
Metrics/MethodLength:
  Include: ["app/controllers/*"]
  Max: 20
Metrics/AbcSize:
  Include: ["app/controllers/*"]
  Max: 30
Metrics/ClassLength:
  Max: 150
Metrics/BlockLength:
  ExcludedMethods: ['describe']

Style/Documentation:
  Enabled: false
Style/ClassAndModuleChildren:
  Enabled: false

Layout/AlignHash:
  EnforcedColonStyle: key
Layout/ExtraSpacing:
  AllowForAlignment: false
Layout/MultilineMethodCallIndentation:
  Enabled: true
  EnforcedStyle: indented
```

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team 

### Only that the files need not be in `tic-tac` folder again.
-  Let's move the files into the root folder. Most probably `OOP-Ruby`
- Also, let's add appropriate linter files. (see comments below)
- [OPTIONAL FOR NOW] Add some details about the project on the readme.

##### See [this review](https://github.com/Elukoye/tic-tac-toe/pull/1#pullrequestreview-280644051) for more details.

#### Please message me on slack if you having issues with the review.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
#### `REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml). :point_down: 

``` yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```
- Also, add the `.rubocop.yml` file, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)

> Remember that `.stickler.yml` and `.rubocop.yml` file names start with a **dot**

**Comment by IBUNHABIBU** [comment]
We have made changes for your guidelines

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+linters:
+   rubocop:
+       display_cop_names: true
+       fixer: true
```
Comment (bolah2009) [inline .rubocop.yml]: ### `REQUIRED`

The `.rubocop.yml` is **still** having the wrong content. Use this instead. You can find the file [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)

```yml
AllCops:
  Exclude:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
  DisplayCopNames: true

Metrics/LineLength:
  Max: 120
Metrics/MethodLength:
  Include: ["app/controllers/*"]
  Max: 20
Metrics/AbcSize:
  Include: ["app/controllers/*"]
  Max: 30
Metrics/ClassLength:
  Max: 150
Metrics/BlockLength:
  ExcludedMethods: ['describe']

Style/Documentation:
  Enabled: false
Style/ClassAndModuleChildren:
  Enabled: false

Layout/AlignHash:
  EnforcedColonStyle: key
Layout/ExtraSpacing:
  AllowForAlignment: false
Layout/MultilineMethodCallIndentation:
  Enabled: true
  EnforcedStyle: indented
```

**Review by bolah2009 — CHANGES_REQUESTED** [review]
> We have made changes for your guidelines

Yes, the files are moved to the root folder. :ok_hand: 

The `.rubocop.yml` and `.stickler.yml` are still having the wrong files.... :confused: 

See [this comment on  `.rubocop.yml`](https://github.com/Elukoye/OOP-Ruby/pull/1#discussion_r319138357) file and [this comment on `.stickler.yml`](https://github.com/Elukoye/OOP-Ruby/pull/1#pullrequestreview-281554132).


Fix this and submit another CRR... :smile:

**Comment by Elukoye** [comment]
corrected

**Review by maelfosso — APPROVED** [review]
# Good job :+1: 

You can merge into the `develop` branch
Good Luck with the next milestone.

**Pull request merged**